prog-o-meter.py

```diff
@@ -144,13 +166,115 @@ def add_day(self):
         self.completion_date = self.get_completion_date(self.days_remaining)
         self.canvas.itemconfig(self.rectangle_list[self.days-1], fill = "green")
         update_days_file(self.filename, self.days)
+        self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining),  " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
+        if self.days >= self.GOAL:  # Disable add_day_button if goal have been reached
+            self.add_day_button.config(state = "disabled")
+            self.add_multidays_button.config(state = "disabled")
+        if self.days > 0 and self.whoops_button["state"] == "disabled":  # Enable whoops_button if days of progress is > 0
+            self.whoops_button["state"] = "normal"
+    def delete_day(self):
+        """Fill last blue rectangle in prog-o-meter with white color, to represent deleting an extra day added by mistake.
+           Callback function to whoops_button. Fills out one more rectangle (most right-ward blue rectangle) with white color, to represent deleting an extra day added by mistake.
+        """
+        self.days -= 1
+        self.days_remaining = self.GOAL - self.days
+        self.completion_date = self.get_completion_date(self.days_remaining)
+        self.canvas.itemconfig(self.rectangle_list[self.days], fill = "white")
+        update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
-        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
-
+        if self.days <= 0:  # Disable Whoops button if days become 0
+            self.whoops_button.config(state = "disabled")
+        # Enable add_day_button if days of progress is < goal
+        if self.add_day_button["state"] == "disabled":
+            self.add_day_button["state"] = "normal"
+        if self.add_multidays_button["state"] == "disabled":
+            self.add_multidays_button["state"] = "normal"
+    def add_multidays(self, days, goal, filename, username):

```

Comment:
remember that you docstring for a function that takes arguments should have an args list, so in this case:
args:
    days: [description of what should be passed, fx. number of days the user have already completed]
    goal: [description of what should be passed]
    filename: [description of what should be passed]
    username: [description of what should be passed]

You can take a look at the docstrings for some of the other functions that takes arguments for some inspiration :)